src/massive/munit/client/PrintClient.hx

```diff
@@ -27,13 +27,9 @@
 ****/
 
 package massive.munit.client;
-import massive.munit.client.PrintClientBase;
-import massive.munit.AssertionException;
-import massive.munit.ITestResultClient;
-import massive.munit.TestResult;
-import massive.munit.util.MathUtil;
+import js.Browser;
```

Comment:
It looks like this class and the other one touched need to compile across target so you can't access the js package here. I'd inline the access like the previous version.

Also is this change backwards compatible? If not it would require a major version bump which I'd like to avoid. Might be worth wrapping the change in version specific flags to maintain backwards compatibility if it is an issue.